### PR TITLE
New ESMF landing page structure

### PR DESCRIPTION
**Pull request by anastascho**
This is the current status of the ESMF landing page structure.
Please review it, but keep in mind that this s still in progress.

### CODE REVIEW

**Comment by anastascho** [comment]
Here you can find a snapshot of the current state of the landing page:

http://fe0vm2112.de.bosch.com/doku/bosch-semantic-stack/ESMF/site/site/esmf-documentation/index.html

File: documentation/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -1,92 +1,183 @@
 = Eclipse Semantic Modeling Framework
-:page-layout: tiles
+:page-layout: sections
```
Comment (michelu89) [inline documentation/modules/ROOT/pages/index.adoc:2]: Can we name it what it is: landing-page-front?
When its possible.

File: documentation/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -1,93 +1,178 @@
...
+with SAMM to base APIs & UIs + 
+on semantic information.
+++++
+ <button type="button" class="get-started-button"><a href="https://eclipse-esmf.github.io/ame-guide/introduction.html" target="_blank">Aspect Model Editor</a></button> 
+++++
+======
+
+[.sectionWhite-right]
+======
+image::ESMF-KeyVisual-Car-Robot.png[esmf-intro-book, window=_blank,opts=nofollow]
```
Comment (aniluede) [inline documentation/modules/ROOT/pages/index.adoc]:
```suggestion
image::ESMF-KeyVisual-Car-Robot.png[esmf-intro-book]
```

Comment (aniluede) [reply documentation/modules/ROOT/pages/index.adoc]: (No link, hence no link attributes needed)

File: documentation/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -1,93 +1,178 @@
...
+
+[.sectionWhite-left]
+======
+
+[.sectionWhite-heading-blue]
+Model aspects of digital twins +
+with SAMM to base APIs & UIs + 
+on semantic information.
+++++
+ <button type="button" class="get-started-button"><a href="https://eclipse-esmf.github.io/ame-guide/introduction.html" target="_blank">Aspect Model Editor</a></button>
```
Comment (aniluede) [inline documentation/modules/ROOT/pages/index.adoc:167]: On the page, this one renders to lead to the SAMM spec instead of to the AME docs.

Comment (aniluede) [reply documentation/modules/ROOT/pages/index.adoc:167]: grafik

File: documentation/modules/ROOT/pages/index-tiles.adoc
Comment (aniluede) [inline documentation/modules/ROOT/pages/index-tiles.adoc:1]: this file is not needed anymore, right?

File: documentation/modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -1,93 +1,178 @@
...
+image::ESMF-SAMM-icon.png[xref=samm-specification:ROOT:index.adoc, window=_blank,opts=nofollow]
+
+[.title][.link]
+xref:samm-specification:ROOT:index.adoc[Semantic Aspect Meta Model, window=_blank,opts=nofollow]
+
+======
 
 [.text]
-Understand SAMM to describe domain semantics for your assets in Aspect Models. SAMM is expressive and fosters reusability.
+The Semantic Aspect Meta Model allows easy construction of models that semantically describe domain data.
```
Comment (aniluede) [inline documentation/modules/ROOT/pages/index.adoc]: Please revert to original text

File: documentation/modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -1,93 +1,178 @@
...
+
+image::ESMF-AspectModel-icon.png[xref=ame-guide:ROOT:introduction.adoc, window=_blank,opts=nofollow]
+
+[.title][.link]
+xref:ame-guide:ROOT:introduction.adoc[Aspect Model Editor, window=_blank,opts=nofollow]
+======
 
 [.text]
-Create your Aspect Models graphically and learn SAMM on the go. Generate downstream artifacts, such as OpenAPI specs.
+Create your Aspect Models graphically and learn SAMM on the go. Create downstream artefacts e.g. OpenAPI spec.
```
Comment (aniluede) [inline documentation/modules/ROOT/pages/index.adoc]: Please revert to original text

File: documentation/modules/ROOT/pages/index.adoc [hunk 6]
```diff
@@ -1,93 +1,178 @@
...
+
+image::ESMF-SAMM-CLI.png[xref=esmf-developer-guide:tooling-guide:samm-cli.adoc, window=_blank,opts=nofollow]
+
+[.title][.link]
+xref:esmf-developer-guide:tooling-guide:samm-cli.adoc[SAMM Command Line Interface, window=_blank,opts=nofollow]
+======
 
 [.text]
-Use common features command-line-based: validate Aspect Models, generate artifacts, such as OpenAPI specs.
+Use common features command line based: Aspect Model validation & artefact generation.
```
Comment (aniluede) [inline documentation/modules/ROOT/pages/index.adoc]: Please revert to original text

File: documentation/modules/ROOT/pages/index.adoc [hunk 7]
```diff
@@ -1,93 +1,178 @@
...
+
+image::ESMF-JAVA-icon.png[xref=esmf-developer-guide:tooling-guide:java-aspect-tooling.adoc, window=_blank,opts=nofollow]
+
+[.title][.link]
+xref:esmf-developer-guide:tooling-guide:java-aspect-tooling.adoc[Java SDK, window=_blank,opts=nofollow]
+======
 
 [.text]
-Load and validate Aspect Models programmatically and generate code to create aspect-based APIs.
+Create Aspect APIs for software solutions. The SDK supports by understanding Aspect Models and generating Java code & artefacts.
```
Comment (aniluede) [inline documentation/modules/ROOT/pages/index.adoc]: Please revert to original text

File: documentation/modules/ROOT/pages/index.adoc [hunk 8]
```diff
@@ -1,93 +1,178 @@
...
+======
+
+image::ESMF-Py-icon.png[xref=python-sdk-guide:ROOT:index.adoc, window=_blank,opts=nofollow]
+
+[.title][.link]
+xref:python-sdk-guide:ROOT:index.adoc[Python SDK, window=_blank,opts=nofollow]
+======
 [.text]
-Load Aspect Models programmatically and generate code for software solutions or analytics use cases.
+Create Aspect APIs in Python for software solutions and analytics use cases.
```
Comment (aniluede) [inline documentation/modules/ROOT/pages/index.adoc]: Please revert to original text

File: documentation/modules/ROOT/pages/index.adoc [hunk 9]
```diff
@@ -1,93 +1,178 @@
...
+======
+
+image::ESMF-JS-icon.png[xref=js-sdk-aml-guide:ROOT:index.adoc, window=_blank,opts=nofollow]
+
+[.title][.link]
+xref:js-sdk-aml-guide:ROOT:index.adoc[JavaScript SDK, window=_blank,opts=nofollow] 
+======
 [.text]
-Build UIs better and faster. This SDK loads Aspect Models and generates UI code, such as tables that unite aspect data and model.
+Create UIs better and faster. This SDK loads Aspect Models and creates UI code e.g. tables that unite Aspect data & model.
```
Comment (aniluede) [inline documentation/modules/ROOT/pages/index.adoc]: Please revert to original text

**Review by aniluede — CHANGES_REQUESTED** [review]

File: documentation/modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (aniluede) [reply documentation/modules/ROOT/pages/index.adoc:2]: Currently not, took a note for next year, thx

File: documentation/modules/ROOT/pages/index.adoc (hunk 3 above)
Comment (aniluede) [reply documentation/modules/ROOT/pages/index.adoc:167]: Already tackled, works now

**Review by aniluede — APPROVED** [review]

**Review by aniluede — APPROVED** [review]
LGTM

**Pull request merged**